index.twig

```diff
@@ -89,10 +89,6 @@
         {% if section.description %}
           <div class="kss-description">
             {{ section.description|raw }}
-
-            {% if section.markupFile %}
-              <p>Twig: <code>{{ section.markupFile }}</code></p>
```

Comment:
Oh sorry I meant remove it from the <summary> element, not here. Since the markup block is optional, if it's turned off we won't have the files reference anywhere and that's the scenario where this line is most important.